Opserver/Views/Dashboard/Node.Services.cshtml

```diff
@@ -24,6 +28,19 @@ Last Updated @(i.LastSync?.ToRelativeTime() ?? "Unknown")" data-val="@(i.Id)">
                 </td>
                 <td class="text-wrap">@i.Description</td>
                 <td>@i.State</td>
+                @if (Current.IsInRole(StackExchange.Opserver.Models.Roles.GlobalAdmin))
```

Comment:
Should be the same check as above...but there's no reason to do the check more than once, I'd just do it once in a `@{ }` block up top.